modules/apps/frontend-js/frontend-js-web/test/liferay/util/add_params.ts

```diff
@@ -65,9 +65,7 @@ describe('Liferay.Util.addParams', () => {
 	});
 
 	it('throws an error if not passed params as an object or a string', () => {
-		expect(() => addParams(1)).toThrow(TypeError);
-
-		expect(() => addParams(['one', 'two'])).toThrow(TypeError);
+		expect(() => addParams(1 as any, sampleUrl)).toThrow(TypeError);
```

Comment:
Another annoyance of keeping runtime type guards around... in tests, to test them you need `foo as any` :)

Funny enough, the second test was passing because the implementation throws `TypeError` from 2 different code branches, not because `typeof [] !== 'object'` 😉

Reply:
I think that if we keep the runtime assertions, then we'd likely want to keep the tests as well?

https://github.com/liferay-frontend/liferay-portal/pull/1379#discussion_r694257114

Following the same reasoning, there's no guarantee people will call this from TS so chances are we won't be testing types?